module/Entra/Microsoft.Entra/Users/Remove-EntraUserSponsor.ps1

```diff
@@ -0,0 +1,29 @@
+# ------------------------------------------------------------------------------ 
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  
+#  Licensed under the MIT License.  See License in the project root for license information. 
+# ------------------------------------------------------------------------------ 
+function Remove-EntraUserSponsor {
+    [CmdletBinding(DefaultParameterSetName = '')]
+    param (
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, HelpMessage = "The unique identifier (User ID) of the user whose sponsor you want to remove.")]
+        [System.String] $UserId,
+
+        [Alias('DirectoryObjectId')]
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, HelpMessage = "The User Sponsor ID to be removed.")]
+        [System.String] $SponsorId
+    )
+
+    PROCESS {
+        $customHeaders = New-EntraCustomHeaders -Command $MyInvocation.MyCommand
+        $params = @{}
+        
+        $params["Uri"] = "https://graph.microsoft.com/v1.0/users/$UserId/sponsors/$SponsorId/`$ref"
+        $params["Method"] = "DELETE"
+
+        Write-Debug("============================ TRANSFORMATIONS ============================")
+        $params.Keys | ForEach-Object {"$_ : $($params[$_])" } | Write-Debug
+        Write-Debug("=========================================================================`n")
+
+        Invoke-GraphRequest -Headers $customHeaders -Uri $($params.Uri) -Method $($params.Method)
```

Comment:
```suggestion
        $response = Invoke-GraphRequest -Headers $customHeaders -Uri $($params.Uri) -Method $($params.Method)
        $response
```

Reply:
Use this as reference
https://github.com/microsoftgraph/entra-powershell/blob/a8872f29d7a8ef19aceeb52eaa0a5e86e13a470e/module/Entra/Microsoft.Entra/SignIns/Remove-EntraPolicy.ps1#L41-#L42